data/avro-schema/src/main/resources/avro/net/corda/data/flow/event/external/ExternalEventContext.avsc

```diff
@@ -0,0 +1,18 @@
+{
+  "type": "record",
+  "name": "ExternalEventContext",
+  "namespace": "net.corda.data.flow.event.external",
+  "doc": "The context of an external event. To be included in the payloads sent as part of the external event structure. Used to extract request information on the responding side.",
+  "fields": [
+    {
+      "name": "requestId",
+      "type": "string",
+      "doc": "The request id of the external event. Used for deduplication when processing responses."
+    },
+    {
+      "name": "flowId",
+      "type": "string",
+      "doc": "The request id of the external event. Used for deduplication when processing responses."
+    }
```

Comment:
I assume one of the doc strings is wrong (or you don't need one of them!). I guess it's the flow ID, which we're using to ensure the response gets put back on the right key?

Reply:
Fied.